src/Hintman/Server/.#Auth.hs

```diff
@@ -0,0 +1 @@
+fenx@haskell.4054:1541310951
```

Comment:
is this a test token?

Reply:
No, this is leftover from Spacemacs. Temporary file, accidentally commited :(